swan-cern/files/swan_computing_config.py

```diff
@@ -393,4 +479,11 @@ def computing_modify_pod_hook(spawner, pod):
     return computing_pod_hook_handler.get_swan_user_pod()
 
 
+# Custom configuration options
+# Name of the role that is assigned to participants of events hosted by SWAN
+events_role = get_config('custom.events.role', 'swan-events')
```

Comment:
Why did you declare the vars here instead of inside the method that uses it? It would be cleaner if in the future we remove that method.

Reply:
I followed the same structure as in the other hooks that have custom config. E.g. see https://github.com/swan-cern/swan-charts/blob/master/swan-cern/files/swan_config_cern.py#L232-L234